internal/cloudinit/cloudinit.go

```diff
@@ -165,7 +165,7 @@ touch $HOME/.env
 echo "SKIP_PREPARE_SERVER=true" >> .env;
 
 {{ if .PluginBinaryURI }}
-wget {{ .PluginBinaryURI }}/plugin-{{ .Platform.OS }}-{{ .Platform.Arch }}  -O /usr/bin/plugin
+- 'wget --retry-connrefused --retry-on-host-error --retry-on-http-error=503,404,529 --tries=10 --waitretry=5 -nv {{ .PluginBinaryURI }}/plugin-{{ .Platform.OS }}-{{ .Platform.Arch }}  -O /usr/bin/plugin'
```

Comment:
why this extra single quote?

Reply:
--waitretry=5, is it 5 seconds?

Reply:
yeah waitretry=5 here 5 is in seconds

Reply:
put waitretry as 1, it is a linear backoff